### DISCUSSION TITLE
NuxtJS.org v3 [preview]

### DISCUSSION THREAD

**Discussion by Tahul** [nuxt/nuxt#9535] [Sharing]
Hello everyone 👋 ;

As announced on Twitter, we are rebuilding the current NuxtJS.org website using Docus!

This is a huge step for the whole Nuxt organization, as much as the Docus project, and we're glad to present that preview to you, our community!

This new website will mark the beginning of the transition between Nuxt 2 and Nuxt 3 documentations, and will be the place to watch out for upcoming releases!

As it is built with Docus, this is a first glance at what this new project will be capable of, we can't wait to get your feedbacks on this! 😄

You can visit it here:
https://preview.nuxtjs.org

Login: `nuxt`
Password: `nuxt`

Contributions are more than welcome, if you spot a bug feel free to fork, and make a PR onto `preview` branch.

Otherwise, you can also create issues about it, without forgetting to mark that it targets that new preview.

This thread is also a great place to discuss improvements and sharing your feedbacks for this website, before creating an issue.

A link to the branch:
https://github.com/nuxt/nuxtjs.org/tree/preview

Thanks to the whole team that works on the project:
@Atinux @clemcode @bdrtsky @pi0 @farnabaz @Flosciante @danielroe @alexchopin

**Comment by codingcn** [comment]
Please give the public beta plan of nuxt3 again

**Reply by codingcn** [reply]
I already know, August 2021  Public beta.

**Reply by codingcn** [reply]
The public beta plan has changed again. It's September

**Comment by kissu** [comment]
So, is this somehow secret (not announced on Twitter yet) until August? 😁

**Reply by bdrtsky** [reply]
I guess public beta will be announced, right now it's private.

**Comment by btkostner** [comment]
This looks good! Does this mean Docus is going to go public soon as well? I'd love to help out with some issues I've already found.

**Reply by atinux** [reply]
Docus will soon be open source indeed, planned this month!

**Reply by mmeester** [reply]
What is Docus? 🤓

**Reply by kissu** [reply]
The new Documentation framework, developed by Nuxt. 👍🏻

**Reply by bdrtsky** [reply]
It's actually much much more than only documentation framework, but let's wait for official info 🤣

**Reply by Stun3R** [reply]
Docus is really insane! I took a look about it and I really like it 💘 Currently using it for some projects with old version, I can't wait the official release ⏳

**Reply by atinux** [reply]
We are currently pushing it to August or September to be based on Nuxt 3 for the public release.

**Reply by Miguelklappes** [reply]
> 
> 
> We are currently pushing it to August or September to be based on Nuxt 3 for the public release.

Regarding Nuxt 3, in [preview.nuxtjs.org](https://preview.nuxtjs.org) it said public beta will be available "in August 2021", now it says "During August 2021". Does that mean Nuxt 3 (public beta) will get released in September and not in August?
![imagen](https://user-images.githubusercontent.com/74425206/127546417-9b4acf3c-1c50-4ed9-ada2-36eaefea1e4d.png)

**Reply by Miguelklappes** [reply]
Well:
![imagen](https://user-images.githubusercontent.com/74425206/127727473-262c5744-327e-4b09-bd1f-1e9284519d6c.png)

**Comment by dacsang97** [comment]
Congrats Nuxt team. But, I don't see Vue Composition API, how about it in Nuxt 3?

**Reply by agnelnieves** [reply]
Hey @dacsang97 It's being mentioned on [Nuxt3's](https://preview.nuxtjs.org/v3) page: 
```
Use the Composition API and Nuxt3's composables for true code re-usability.
```

**Reply by bdrtsky** [reply]
Composition API will be definitely supported

**Comment by agnelnieves** [comment]
Niceee! Love the new layout and even the logo! Is there any plans for locales? I'd love to contribute to the spanish docs!

**Reply by Kapcash** [reply]
There's French available already, so I guess Spanish will come soon! I'm ready to translate as well 👍

**Reply by clemcode** [reply]
Hi,

we'll get in touch with the translations projects lead very soon, and release a process for translation using Docus.

Thanks for your support and participation, expect to hear from us!

**Comment by alvarosabu** [comment]
Amazing work, just amazing 👏🏻 looking forward to test that beta

**Comment by paulmelero** [comment]
Great job, Nuxt team!!!

🎉🎉🎉 Really excited about this! As I bet you all are too. 🎉🎉🎉

A couple of observations on the new docs design:

- In the sidebar, the _partners_ and the _ToC_ are too similar or it's not sufficiently differentiated; could cause a lot of missclicks: (I don't know if this is a temporary implementation 😉)

![image](https://user-images.githubusercontent.com/6775220/125573101-ec0322a2-0093-4e96-8902-304fcfa225d0.png)

- The "contributors" block is missing 😢. That was a really nice feature. Again, I don't know if you're still in the middle of porting things to the new docs, I assume so. ❤️

**Reply by Kapcash** [reply]
Contributors appear on the team page: https://preview.nuxtjs.org/team 😁
I guess you liked that the list was present in the footer of every page?

**Reply by kissu** [reply]
Yep, he is probably talking about the footer one. 😉

**Reply by paulmelero** [reply]
Yes, the footer one had information on the people who contributed to each docs page. Webpack docs do the same thing.

**Reply by clemcode** [reply]
Hi,

indeed the contributors block has not yet been migrated, but it's definitely on the roadmap and will be there for official release.

Regarding the ToC and partners block we'll find a way to differentiate it more.

Thanks for your feedbacks!

**Reply by Kapcash** [reply]
Oh nice, I didn't even spotted it was only contributors to the current page! Good thing it will be added then :D

**Comment by davidhellmann** [comment]
Two feedbacks:

1. I don't like this mouse following diamonds. Feels a bit old fashioned. Not the diamonds itself but how they used. Maybe it needs a bit more  IN YOUR FACE in this first section.
![image](https://user-images.githubusercontent.com/764292/125581287-5fc82ecd-f357-403c-bbc0-3a3257fdb4aa.png)

2. I like samples like this but again it's a bit unenthusiastic in this state. stripe.com shows how stuff like this have to look like. 
![image](https://user-images.githubusercontent.com/764292/125581747-93876101-db30-4ff2-bd20-9695256dcccd.png)

Cheers

**Reply by Kapcash** [reply]
I think this is a matter of taste, not really something we can really criticize. I do like the diamonds moving along with the cursor. About the small animation, well I do like it as well. It's not as impressive as it could be but hey, they're doing there best here 😁

**Reply by davidhellmann** [reply]
Sure, valid point and yes it's a personal opinion. But for a framework with this importance I would have expected a bit more of a page here and there. I don't choose nuxtjs based on their website but I don't do that with a payment processor either. Still, stripe.com leaves a mark in my mind. But design feedback is always difficult anyway :)

**Reply by bdrtsky** [reply]
I totally agree with both suggestions. But this is actually a transition website for Nuxt 2 docs. It will be definitely better when Nuxt 3 will be ready to release along with completed redesign.

**Comment by sibbng** [comment]
Why some components imports Composition APIs from `@vue/composition-api` and some of them from `@nuxtjs/composition-api`?

**Comment by emilsgulbis** [comment]
Animation for this should start only when the section visible on viewport, in my opinion. Otherwise, it shows the finished index.vue file and it makes no sense
<img width="1312" alt="Screenshot 2021-07-14 at 12 07 29" src="https://user-images.githubusercontent.com/3534160/125595812-5d0aaf4a-3128-429d-9bdb-63a08ee3491c.png">

**Reply by bdrtsky** [reply]
Valid, right now it's pure CSS animation, so no way to pause it

**Reply by alvarosabu** [reply]
Use an `intersectionObserver` and add the class with the animation so it can start  ;)

**Reply by clemcode** [reply]
Hi,

thanks for your feedback, we'll add it before official release!

**Reply by bdrtsky** [reply]
@alvarosaburido great suggestion. Tho, I counted 31 `@keyframes` in those animation components, so my guess it's not that simple 😀 Anyway, I am pretty sure this animation will be written from scratch at some point.

**Comment by rishi-raj-jain** [comment]
Looks so clean ✨
And those fonts 👀🚀

**Comment by JHerwig** [comment]
How about add another item in the head "Help" which refers to this [discussions](https://github.com/nuxt/nuxt.js/discussions/) page? We should lead new people to use github instead of discord.

**Reply by kissu** [reply]
I'll make some ad for the discussions when possible (at a conference)!

**Comment by CavalcanteLeo** [comment]
waiting for a lifetime

**Reply by kissu** [reply]
Coming soon, keep the faith! 😸 
![image](https://user-images.githubusercontent.com/5133074/128319571-7722e753-7e55-4f30-856f-1cf80db9ef39.png)

**Reply by rickyvadala** [reply]
HI! @kissu 

I have to start a new project in October and I'm considering starting it with the public beta but I couldn't find it.

The web says the public was going to be released in September and the Stable on Q4, is that plan still in effect?

**Reply by kissu** [reply]
@rickyvadala more info during Nuxtnation on Wednesday. 😉

**Reply by mmeester** [reply]
haha such a sneaky way to get a scoop 🤣

**Reply by atinux** [reply]
We finally decided for a date (not easy for such a big project), it will be announced tomorrow :)

**Reply by kissu** [reply]
> We finally decided for a date (not easy for such a big project), it will be announced tomorrow :)

Not sure that people will be able to wait until today/tomorrow evening. 😂

**Reply by rickyvadala** [reply]
> > We finally decided for a date (not easy for such a big project), it will be announced tomorrow :)
> 
> Not sure that people will be able to wait until today/tomorrow evening. 😂

404 sleeping tonight NOT FOUND
200 anxiety OK

**Comment by ADZG** [comment]
I want the menu bar on the left to be fixed, because the menu bar on the right is too long,

Or add a back to top button in the lower right corner

**Comment by shaunc** [comment]
Seems to be down?

![image](https://user-images.githubusercontent.com/193170/135482610-c24dfc6d-cda5-468d-87be-990a793677c6.png)

**Reply by kissu** [reply]
Check the official website. 😉
https://nuxtjs.org/v3

**Reply by shaunc** [reply]
Ah ... ok -- saw that but didn't realize I could scroll down.... :)

**Reply by kissu** [reply]
No issues. 😉

**Reply by shaunc** [reply]
... hmm... not much info though... seems like an ad. I want doc... do I have to wait until actual release even to read about it (in depth)?

**Comment by rahulkumarsingh73690** [comment]
What about a stable release? (Any timeline)

**Reply by JokerMartini** [reply]
I'm wondering the same thing, any idea of timeline on stable release?

**Reply by Miguelklappes** [reply]
Q4 2022

**Reply by kissu** [reply]
You could probably get more info on the VueJS Nation, stay tuned! ✨ 
https://vuejsnation.com/

**Reply by JokerMartini** [reply]
darn that is a bummer and quite a ways away. looking forward to it though.

**Reply by kissu** [reply]
Also, it's probably not easy to give an estimate.

Especially when you always have a lot of nice ideas coming up and some unexpected things popping from time to time.
I have trouble giving a proper deadline for a 2-week work. I just don't know how it is feasible to achieve one for a far bigger and more complex task.

So let's try to wait for some incoming news. 🤗
